direct/src/directtools/DirectManipulation.py

```diff
@@ -1,10 +1,48 @@
+import math
+from panda3d.core import (
+    BitMask32,
+    BoundingHexahedron,
+    CSDefault,
+    Mat4,
+    NodePath,
+    Point3,
+    VBase3,
+    VBase4,
+    Vec3,
+    decomposeMatrix,
+    deg2Rad,
+)
 from direct.showbase.DirectObject import DirectObject
 from direct.showbase.MessengerGlobal import messenger
-from .DirectGlobals import *
-from .DirectUtil import *
-from .DirectGeometry import *
+from .DirectGlobals import (
+    EDIT_TYPE_UNEDITABLE,
+    EDIT_TYPE_UNMOVABLE,
+    EDIT_TYPE_UNROTATABLE,
+    EDIT_TYPE_UNSCALABLE,
+    MANIPULATION_MOVE_DELAY,
+    ORIGIN,
+    SKIP_BACKFACE,
+    SKIP_CAMERA,
+    SKIP_HIDDEN,
+    SKIP_UNPICKABLE,
+    SKIP_WIDGET,
+    X_AXIS,
+    Y_AXIS,
+    Z_AXIS,
+    LE_showInOneCam,
+)
```

Comment:
This is a bit awkward, can we just prefix it, like `from . import DirectGlobals as DG`?